scm/driver/bitbucket/webhook.go

```diff
@@ -902,3 +1045,50 @@ func convertPrCommentHook(src *prCommentHook) *scm.IssueCommentHook {
 	}
 	return &dst
 }
+
+func convertBitbucketHook(src *pipelineHook) *scm.PipelineHook {
+	namespace, name := scm.Split(src.Repository.FullName)
+
+	return &scm.PipelineHook{
+		Repo: scm.Repository{
+			ID:        src.Repository.UUID,
+			Namespace: namespace,
+			Name:      name,
+			Clone:     src.Repository.Links.HTML.Href, // Assuming HTML link as Clone URL
+			CloneSSH:  "",                             // Bitbucket webhook does not provide SSH URL directly
+			Link:      src.Repository.Links.Self.Href,
+			Branch:    src.CommitStatus.RefName,
+			Private:   src.Repository.IsPrivate,
+		},
+		Commit: scm.Commit{
+			Sha:     src.CommitStatus.Commit.Hash,
+			Message: src.CommitStatus.Commit.Message,
+			Author: scm.Signature{
```

Comment:
the commit author may not be the same as the actor (person who triggered a pipeline)

Reply:
updated in latest changes